README.md

```diff
@@ -487,24 +470,24 @@ curl 'http://localhost:3000/api/users' \
 }'
 ```
 
-# Frontend with React & MUI
+# Frontend with React and MUI
 
-Now that our backend is configured and running, we can now use the same
-schemas as defined earlier to quickly hatch our frontend. The first
+Now that our backend is configured and running we can use the same
+schemas that we defined earlier to quickly _hatch_—see what we did there?—our frontend. The first
```

Comment:
There's so much jargon in code that it's hard to know what's standard and what's a one-off pun. I recommend against puns/slang like this unless we're explicit about what we're doing.

Reply:
Too much lampshade.  I think we could just quote "hatch" here and it would convey that it's referring back to the name.

Reply:
i agree with Brad, `"hatch"` instead of just `hatch`

Reply:
Done!